protocole/transaction.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
deleted file: are you sure ?

Reply:
There was a git failure 😅 

It was the reason the ci failed

Reply:
It was in the wrong folder 😅